src/Aquifer.Common/Utilities/BibleTextUtilities.cs

```diff
@@ -0,0 +1,140 @@
+using System.Collections.ObjectModel;
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Common.Utilities;
+
+public static class BibleTextUtilities
+{
+
+    public static async Task<List<BibleText>> GetVersificationForBibleId(int toBibleId, int startVerseId, int endVerseId,
+        ICachingVersificationService versificationService, AquiferDbContext dbContext, CancellationToken ct)
+    {
+        const int fromBibleId = 1;
+        var bibleTexts = new List<FlatBibleText>();
+        try
+        {
+            var verses = await dbContext.Verses.Where(v => v.Id >= startVerseId && v.Id <= endVerseId).Select(v => v.Id).ToListAsync(ct);
+            var fromBibleToBaseMapping = await versificationService.GetVersificationsByBibleIdAsync(fromBibleId, ct);
+            var toBibleToBaseMapping = await versificationService.GetVersificationsByBibleIdAsync(toBibleId, ct);
+
+            var fromBibleToBibleMapping = GetMergedMappingsOnBaseId(fromBibleToBaseMapping, toBibleToBaseMapping);
```

Comment:
I think these can be handled by VersificationCaching, where we just send it a fromBibleId, toBibleId and get the dictionary back.

Reply:
On second thought, should probably make a `VersificationUtilities` because it only merges what comes out of the cache and doesn't cache the `fromBibleIdToBibleId` map.......